roles/openshift_logging_eventrouter/defaults/main.yaml

```diff
@@ -0,0 +1,9 @@
+---
+openshift_logging_eventrouter_image_prefix: "{{ 'gcr.io/heptio-images/' }}"
+openshift_logging_eventrouter_image_version: "{{ 'v0.1' }}"
+openshift_logging_eventrouter_replicas: 1
+openshift_logging_eventrouter_sink: stdout
```

Comment:
Not sure you saw my previous comment but we should validate the sink values similiar to how we validate the logging log appendar (file vs console) which are the only 2 supported options.

Reply:
thanks! I did miss it but added it now.

Also may be worth mentioning that currently implemented check for `openshift_logging_es_log_appenders` doesn't appear to be checking values, only size
https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_logging_elasticsearch/tasks/main.yaml#L110-L113

Reply:
Hmm... I thought it was changed to be like: https://github.com/openshift/openshift-ansible/pull/5048/files#diff-a3144accf55f93bdfdac09292c2ac475R105